From 05b1fd974c71041265a862ca3a2ba4fc79e797cc Mon Sep 17 00:00:00 2001
From: artoo <artoo@artixlinux.org>
Date: Tue, 29 Aug 2023 16:13:30 +0200
Subject: [PATCH] meson-libexec-dir.patch

---
 meson.build       | 2 +-
 meson_options.txt | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 3e94ce5d..80d067cc 100644
--- a/meson.build
+++ b/meson.build
@@ -102,7 +102,7 @@ if os == 'Linux' and libexecdir == 'libexec'
   libexecdir = 'lib'
 endif
 libexecdir = rootprefix / libexecdir
-rc_libexecdir = libexecdir / 'rc'
+rc_libexecdir = libexecdir / get_option('librcdir')
 rc_bindir = rc_libexecdir / 'bin'
 rc_sbindir = rc_libexecdir / 'sbin'
 rc_shdir = rc_libexecdir / 'sh'
diff --git a/meson_options.txt b/meson_options.txt
index 2c74152e..d2f67e4f 100644
--- a/meson_options.txt
+++ b/meson_options.txt
@@ -37,3 +37,5 @@ option('termcap', type : 'combo',
   description : 'the termcap library to use')
 option('zsh-completions', type : 'boolean',
   description : 'install zsh completions')
+option('librcdir', type : 'string', value : 'rc',
+  description : 'default location of rc libexec dir')
