From 6f9e4c6b4bebebad2f00d1c19bf1f93c707d9a09 Mon Sep 17 00:00:00 2001
From: artoo <artoo@artixlinux.org>
Date: Sun, 14 Apr 2024 10:52:40 +0200
Subject: [PATCH] artix rc.conf

---
 etc/rc.conf | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/etc/rc.conf b/etc/rc.conf
index d359eb9e..d44b6da9 100644
--- a/etc/rc.conf
+++ b/etc/rc.conf
@@ -19,7 +19,7 @@
 # If not specified we use $SHELL, otherwise the one specified in /etc/passwd,
 # otherwise /bin/sh
 # Linux users could specify /sbin/sulogin
-#rc_shell=/bin/sh
+rc_shell=/usr/bin/sulogin
 
 # Do we allow any started service in the runlevel to satisfy the dependency
 # or do we want all of them regardless of state? For example, if net.eth0
@@ -48,7 +48,7 @@
 # /var/log/rc.log
 # NOTE: Linux systems require the devfs service to be started before
 # logging can take place and as such cannot log the sysinit runlevel.
-#rc_logger="NO"
+rc_logger="YES"
 
 # Through rc_log_path you can specify a custom log file.
 # The default value is: /var/log/rc.log
@@ -89,7 +89,7 @@
 # There variables are shared between many init scripts
 
 # Set unicode to NO to turn off unicode support for keyboards and screens.
-#unicode="YES"
+unicode="YES"
 
 # This is how long fuser should wait for a remote server to respond. The
 # default is 60 seconds, but  it can be adjusted here.
@@ -199,7 +199,7 @@ rc_tty_number=12
 # cgroups version 1 on /sys/fs/cgroup.
 # "legacy" mounts cgroups version 1 on /sys/fs/cgroup
 # "unified" mounts cgroups version 2 on /sys/fs/cgroup
-#rc_cgroup_mode="unified"
+rc_cgroup_mode="unified"
 
 # This is a list of controllers which should be enabled for cgroups version 2
 # when hybrid mode is being used.
